src/puppetlabs/trapperkeeper/services_internal.clj

```diff
@@ -139,6 +139,7 @@
   (let [f (first forms)]
     (cond
       (symbol? f) (merge {:service-protocol-sym f} (validate-deps-form! (rest forms)))
+      (map? f) (merge {:service-protocol-sym nil} (validate-deps-form! forms))
```

Comment:
Just to verify, there is currently no other legal syntax where a map would end up in this position, except for the optional dependencies case.  Correct?